starport/pkg/events/events_test.go

```diff
@@ -0,0 +1,191 @@
+package events
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestBus_Send(t *testing.T) {
```

Comment:
```suggestion
func TestBusSend(t *testing.T) {
```

Can we avoid using lowers cases?